operations/messaging-and-math/README.md

```diff
@@ -1,14 +1,14 @@
 ---
-description: 1% DRAFT. Overview of Marketing department and where to find materials
+description: Overview of Marketing department and where to find materials
 ---
 
 # Messaging and Math
 
 ## Why "Messaging and Math"?
 
-Rather than always call a department "Marketing"--which is an extremely broad term that involves every interaction prospects, customers, our broader [community](../../contributors/contributors/community.md) and the industry has with our brand, product and company--we use the short-hand "Messaging and Math" to refer to the core functions of the department:
+Rather than always call a department "Marketing"--which is a broad term that involves every interaction prospects, customers, our broader [community](../../contributors/contributors/community.md) and the industry has with our brand, product and company--we use "Messaging and Math" to refer to the core functions of the department:
```

Comment:
```suggestion
Rather than always call a department "Marketing"--which is a broad term that involves every interaction prospects, customers, our broader [community](../../contributors/contributors/community.md) and the industry has with our brand, product and company - we use "Messaging and Math" to refer to the core functions of the department:
```